src/UniversalTelegramBot.cpp

```diff
@@ -638,5 +688,15 @@ bool UniversalTelegramBot::sendChatAction(String chat_id, String text)  {
     }
   }
 
+  closeClient();
   return sent;
 }
+
+void UniversalTelegramBot::closeClient(){
+#ifdef ESP32
```

Comment:
I know in the telegram chat we talked about making this optional, but could we make it on by default? esp8266 2.4 has been released now so that issue probably won't exist anymore. (I can also test it's impact either way as I have a IDE set up with 2.3 and 2.4)

so maybe have something like 

```
#ifdef DONT_STOP_CLIENT
// don't close client
#else 
// stuff you currently have
#endif

Reply:
Hey, I'm so sorry I hadn't looked at your comment earlier. Is there still something that I need to look at?